doc/WEBUI.md

```diff
@@ -202,6 +202,7 @@ single or double quotes.
   - `netdev`, `networkdevice` look for network devices (firewalls,
     routers, ...).
   - `phonedev` look for telephony devices.
+  - `cpe:[value] | cpe:[field=value](, ...)` look for a given cpe, by value or by field (type, vendor, component or product).
```

Comment:
While this is right, using the same separator between the request type (cpe) and the fields/value seems a little odd to me. Moreover, cpe strings contain multiple ":", and the parser could get messy if we want to keep the "cpe:cpe_string" syntax. That said, if you really prefer the full ":" syntax, I will update the parser accordingly.

What is your preference?